### DISCUSSION TITLE
NFC Extender ( Range increase )

### DISCUSSION THREAD

**Discussion by LLucian97** [community/community#131826] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone. 
I'm sorry if i posting in wrong category.

I need  something to increase NFC range on android With Type -C or similar.
I searching 2 weeks, and i don't find what i want, beacause for security it's only 5 cm.
Can someone do or create this in Privat ?

Thank You.

![Screenshot_20240711_144438_Chrome](https://github.com/community/community/assets/109835374/353c3857-cc6b-4dd1-ad50-743ed13e89c4)
![Screenshot_20240711_144516_Chrome](https://github.com/community/community/assets/109835374/c3d208d0-1320-422a-81ea-fdbec81d3089)

**Comment by paiineli** [comment]
Hi,

You’re in the right place to ask for help! Increasing the NFC range beyond its standard limit can be quite challenging due to built-in security measures. Unfortunately, most NFC technology is designed to operate within a short range (typically around 5 cm) for security reasons.

Here are a few suggestions:

NFC Extenders: Look into NFC extenders or specialized hardware that may increase the effective range. However, keep in mind that they might still be limited by security protocols.

Alternative Technologies: If longer-range communication is essential, consider using other technologies like Bluetooth or RFID, which can operate over greater distances.

Custom Solutions: If you're looking for a private project or custom solution, I recommend reaching out to hardware developers or engineers who specialize in NFC technology. They might be able to create a solution tailored to your needs.

If you have more specific requirements or ideas, feel free to share!

Best,
